tests/console/ant.pm

```diff
@@ -82,7 +82,8 @@ sub run {
     zypper_call 'in ant';
 
     # Set JAVA_HOME to the jdk installation directory
-    assert_script_run("export JAVA_HOME=`update-alternatives --list javac| awk -F 'bin' '{split(\$0, a); print a[1]}'`");
+    assert_script_run("export JAVA_HOME=/usr/lib64/jvm/java");
```

Comment:
I'm really not sure about this arch `lib64` dependent path.

Reply:
ok, I will revert to the regexp method then